random_joke.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask,request
+from clients import JokeApi
+from schemas import MultipleJokesRequestParams
+from pydantic import ValidationError
+
+app = Flask(__name__)
+
+
+@app.route("/random-joke")
+def return_joke():
+    return JokeApi.get_random_joke()
+    
+@app.route("/random-5-jokes")
+def return_five():
+    return JokeApi.five_jokes()        
+
+@app.route("/multi-random-joke")
+def multiple_joke():
+    # Raw query data
+    raw_request = request.args
+    #Input validation usin pydantic
+    try:
+        MultipleJokesRequestParams(**raw_request)
+    except ValidationError as e:
+        return(e.json())
```

Comment:
On this case e will be an instance of the `ValidationError` class. You can actually convert it to a string as [per the docs](https://pydantic-docs.helpmanual.io/usage/validators/) then simply return that string back to the client which will provide them with a good explanation of why it has failed:

So in this case:

```
except ValidationError as exc:
      return str(exc)
```